sentinel-adapter/sentinel-zuul-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/zuul/filters/SentinelZuulPreFilter.java

```diff
@@ -59,15 +60,19 @@
 
     private final int order;
 
-    private final GatewayParamParser<RequestContext> paramParser = new GatewayParamParser<>(
-        new RequestContextItemParser());
+    private final GatewayParamParser<RequestContext> paramParser;
 
     public SentinelZuulPreFilter() {
         this(10000);
     }
 
     public SentinelZuulPreFilter(int order) {
+        this(order, new RequestContextItemParser());
+    }
+
+    public SentinelZuulPreFilter(int order, RequestItemParser<RequestContext> requestItemParser) {
         this.order = order;
+        this.paramParser = new GatewayParamParser<>(requestItemParser == null ? new RequestContextItemParser() : requestItemParser);
```

Comment:
if our requestItemParser is can not be null, I think assert it cannot null is better. because we have provided the default value constructor.